fairlearn/metrics_plots/plot_disparities_in_metric.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from fairlearn.metrics import MetricFrame
+
+
+def plot_disparities_in_metric(metric, y_true, y_pred, sensitive_features, show_plot=True):
+    """Plot disparities in a metric between groups defined by sensitive features.
+
+    This method visualizes the metric values for all groups as identified
+    by sensitive features.
+
+    Parameters
+    ----------
+    metric : callable
+        A metric function that takes arguments `y_true` and `y_pred`
+    y_true : array-like
+        The list of true values
+    y_pred : array-like
+        The list of predicted values
+    sensitive_features : array-like
+        the sensitive attributes
+
+    """
+    # compute
+    metric_frame = MetricFrame(metric, y_true, y_pred,
+                               sensitive_features=sensitive_features)
+
+    # chart text for localization
+    metric_text = metric.__name__.replace('_', ' ')
```

Comment:
we could accept `metric_name` or `x_label`, `y_label` optionally if the `__name__` is not properly set or the user wants an alternative text.

Reply:
Agreed! I added an optional argument `metric_name` and documented that we'll use `__name__` is no metric name is passed.